identity/template_loader.py

```
@@ -0,0 +1,56 @@
+
+    Args:
+        task_type: 任务类型 (chat/code/writing/analyze)
+
+    Returns:
+        模板内容字符串，如果文件不存在则返回 None
+    """
+    template_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'specs')
+    template_file = os.path.join(template_dir, f'{task_type}.md')
+
+    if not os.path.exists(template_file):
+        # 如果特定类型的模板不存在，尝试使用 base.md
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)